src/main/java/org/pac4j/demo/j2e/ui/ProfileView.java

```diff
@@ -55,10 +54,41 @@ public List getProfiles() {
     }
 
 
+    /**
+     * Gets the Google OIDC profile if it exists.
+     *
+     * @return the Google OIDC profile
+     */
+    public GoogleOidcProfile getGoogleOidcProfile() {
```

Comment:
I really liked the `ProfileView` as a generic helper to access the profiles, something I can put in j2e-pac4j v3.0. What about a `getProfile(Class)` method? I found these methods too specific...